webapp/components/wpt-insights.html

```diff
@@ -0,0 +1,68 @@
+<!--
+Copyright 2018 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="info-banner.html">
+<link rel="import" href="product-info.html">
+<link rel="import" href="wpt-flags.html">
+
+<dom-module id="wpt-insights">
+  <template>
+    <style>
+      info-banner {
+        margin: 0;
+      }
+      paper-card {
+        margin: 1em;
+      }
+    </style>
+    <template is="dom-repeat" items="[[insights]]" as="item">
+      <paper-card>
+        <div class="card-content">
+          <h3>[[item.name]]</h3>
+          <info-banner>
+            <a class="query" href="[[item.url]]">
+              [[item.query]]
+            </a>
+          </info-banner>
+          <p>
+            [[item.desc]]
+          </p>
+        </div>
+      </paper-card>
+    </template>
+  </template>
+  <script>
+    /* global ProductInfo, WPTFlags */
+    class Insights extends ProductInfo(WPTFlags(window.Polymer.Element)) {
+      static get is() {
+        return 'wpt-insights';
+      }
+
+      get insights() {
+        const browsers = ['chrome', 'edge', 'firefox', 'safari'];
```

Comment:
nit: Should this be configurable? (I.e., these four strings are the default value on a property.)

Reply:
... ditto for the `insights` property. Should we have a helper that generates a default value so that the owner of this component could, instead, bind a value to `insights`?

Reply:
Eventually :)